src/machines-vs-machines-sfos.cpp

```diff
@@ -76,7 +98,29 @@ int main(int argc, char *argv[])
     qmlRegisterType<TowerProxyModel>("harbour.machines.vs.machines.sfos.Machines", 1, 0, "TowerProxyModel");
     qmlRegisterType<BoardProxyModel>("harbour.machines.vs.machines.sfos.Machines", 1, 0, "BoardProxyModel");
 
+    //qmlRegisterSingletonType<Settings>("harbour.machines.vs.machines.sfos.Machines", 1, 0, "SettingsBackend", createSettings);
+
+    //return SailfishApp::main(argc, argv);
+
+
+    QScopedPointer<QGuiApplication> app(SailfishApp::application(argc, argv));
+
+    app->setApplicationName("harbour-machines-vs-machines-sfos");
+    app->setOrganizationDomain("com.github.mzanetti");
+    app->setOrganizationName("com.github.mzanetti");
+/*
+    QString newConfigDir = QStandardPaths::writableLocation(QStandardPaths::AppConfigLocation);
+    QString configFile = QCoreApplication::applicationName() + ".conf";
+    QSettings global(newConfigDir + "/" + configFile, QSettings::IniFormat);
+*/
```

Comment:
Looks good. perhaps the commented out stuff can even be dropped

Reply:
Yes. That's the 'standard' approach which doesn't apply in this case because of the way the Settings init functions in the settings.h. It took me a while to grok, but with Sailjail it get's tricky. In any case, this method works so the other code can go.